README.rst

```diff
@@ -43,10 +43,93 @@ Zocalo
+
+.. _xia2: https://xia2.github.io/
+.. _fastep: https://github.com/DiamondLightSource/fast_ep
+
+Working with Zocalo
+-------------------
+
+`Graylog <https://www.graylog.org/>`_ is used to manage the logs produced by Zocalo. Once Graylog and the message broker server are running then services and wrappers can be launched with Zocalo. 
+
+Zocalo provides some command line tools. These tools are ``zocalo.go``, ``zocalo.wrap`` and ``zocalo.service``: the first triggers the processing of a recipe and the second runs a command while exposing its status to Zocalo so that it can be tracked. Services are available through ``zocalo.service`` if they are linked through the ``workflows.services`` entry point in ``setup.py``. For example, to start a Schlockmeister service:
+
+.. code:: bash
+
+        $ zocalo.service -s Schlockmeister
+
+.. list-table:: 
+        :widths: 100
+        :header-rows: 1
+
+        * - Q: How are services started?
+        * - A: Zocalo itself is agnostic on this point. Some of the services are self-propagating and employ simple scaling behaviour - in particular the per-image-analysis services. The services in general all run on cluster nodes, although this means that they can not be long lived - beyond a couple of hours there is a high risk of the service cluster jobs being terminated or pre-empted. This also helps encourage programming more robust services if they could be killed.
+
+.. list-table:: 
+        :widths: 100
+        :header-rows: 1
+
+        * - Q: So if a service is terminated in the middle of processing it will still get processed?
+        * - A: Yes, messages are handled in transactions - while a service is processing a message, it's marked as "in-progress" but isn't completely dropped. If the service doesn't process the message, or it's connection to ActiveMQ gets dropped, then it get's requeued so that another instance of the service can pick it up.
+
+Repeat Message Failure 
+----------------------
+
+How are repeat errors handled? This is a problem with the system - if e.g. an image or malformed message kills a service then it will get requeued, and will eventually kill all instances of the service running (which will get re-spawned, and then die, and so forth).
+
+We have a special service that looks for repeat failures and moves them to a special "Dead Letter Queue". This service is called Schlockmeister_, and is the only service at time of writing that has migrated to the public zocalo repository. This service looks inside the message that got sent, extracts some basic information from the message in as safe a way as possible and repackages to the DLQ with information on what it was working on, and the "history" of where the message chain has been routed.
+
+.. _Schlockmeister: https://github.com/DiamondLightSource/python-zocalo/tree/master/zocalo/service
+
+Documentation 
+-------------
+
+Further documentation is available at https://zocalo.readthedocs.io.
 
 Features
```

Comment:
Can remove the pointless `Features` section

Reply:
Done. I'll merge once the checks have passed